jgrapht-io/src/main/java/org/jgrapht/io/DOTExporter.java

```diff
@@ -262,7 +263,7 @@ private void renderAttributes(PrintWriter out, String labelName, Map<String, Att
             }
         }
         if (labelName != null) {
-            out.print("label=\"" + labelName + "\" ");
+            out.print("label=\"" + escapeDoubleQuotes(labelName) + "\" ");
```

Comment:
I think we need to apply this to the user-supplied attributes below, as well.  Right?

Reply:
Oh, looks like you took care of this already.